snake/src/main/scala/io/circe/yaml/parser/parse.scala

```diff
@@ -0,0 +1,8 @@
+package io.circe.yaml.parser
+
+import io.circe.yaml.snake
+import io.circe.{Json, ParsingFailure}
+
+object parse extends Parser {
```

Comment:
Hmm, not sure. I guess it would be closer to the definition of `Parser`. I do want to override the default argument of `v1` from `Function1` so that it looks nicer in IDEs. I'll give this a shot and see how it looks.